tests/test_ensemble.py

```diff
@@ -27,8 +27,10 @@ def test_single_model(self):
 
         m = OneInputOneOutputOneEventLM()
         
-        with self.assertRaises(ValueError):
+        try:
             EnsembleModel([m])
+        except ValueError:
+            self.fail("Error EnsembleModel raised ValueError Unexpectedly")
```

Comment:
```suggestion
        EnsembleModel([m])
```
Try-except not necessary - test would fail if value error is raised